User-Stories/PO-PM-User-Stories/PO-user-stories.md

```diff
@@ -0,0 +1,60 @@
+#Product Owner (PO) / Project Manager (PM) user stories
+We assume this role as a principal role compared to a developer. Thus this player has the right to task the developer (agent) and influences the priotization of the agents work. Typically such a role is a team lead or project manager or in modern words a product owner. For ease of read, we will title the role PO, konwing that this simplification will not cover all details associated with the different role models. But from a compliance perspective the models can be treated as equivalent.
+
+##Responsibilites, tasks and role
+The ```primary goal``` of this role is to deliver a software solution while minimizing all related risks. This may comprise security, export controls or the incompliance with obligations resulting from the application of open source components. 
+
+Thus this role will be ```accountable``` for:
+* Execution of all reuired tasks to provide a risk minimized software solutions
+* Arrange and provide all relevnat tasks to cope with resulting obligations
+* Completeness of files/data/code to be assessed and delivered within the scope of this project
+
+This typically results in the follwing ```tasks```:
+* Ensure code is scanned on a regular basis - optimally as part of the CI/CD-chain
+* Ensure that _all_ components _and_ resources comprised within the solution will be captured/covered by the scans (incl. runtime components)
+* Initiate appropriate measures to cope with obligations and provide associated documentation
+* Initiate measures to cope with violations and warnings
+* Initiate approval requests
+* Escalate questionnable components/licenses to Compliance Manager/OS board
+
+Therefor the most ```critical success factors``` of theis role are:
+* Understanding of the Open Source Policy
+* Willingness to provide a proper and sound documentation
```

Comment:
which kind of documentation has this role to provide

Reply:
If the PO has no stakes in providing a sound documentation about his solution, the whole ambition will fail. that's why I see his willingness as a cirtical success factor.

Reply:
I was not precise - I neant which topics shall the documentation cover - e.g. technical description or complaince documentation or user guide

Reply:
Expanded a little.